client/src/components/content/ModelerSidebar/ModelerSidebar.jsx

```diff
@@ -1,24 +1,21 @@
 /* eslint-disable react/jsx-no-bind */
 import React, { useState, useEffect } from 'react';
-import { Typography, Layout, Space, Button, notification } from 'antd';
+import { Typography, Layout, Space, Button } from 'antd';
 import PropTypes from 'prop-types';
-import { CloudUploadOutlined } from '@ant-design/icons';
 import PropertiesPanel from './PropertiesPanel/PropertiesPanel';
 import styles from './ModelerSidebar.module.css';
-import { fetchTasksFromDB } from '../../../api/applicationAndTaskSelection';
-import getParsedRobotFile from '../../../api/ssot';
-import downloadString from '../../../utils/downloadString';
-import {
-  setSingleParameter,
-  resetRpaApplication,
-  setRpaTask,
-  upsert,
-  getParameterObject,
-  setOutputValueName,
-} from '../../../utils/attributeAndParamUtils';
 import parseSsotToBpmn from '../../../utils/ssotToBpmnParsing/ssotToBpmnParsing';
-import { parseBpmnToSsot } from '../../../utils/BpmnToSsotParsing/BpmnToSsotParsing';
-import corporateDesign from '../../../layout/corporateDesign';
+import {
+  nameChangedHandler,
+  applicationChangedHandler,
+  taskChangedHandler,
+  handleInputParameterChange,
+  handleOutputVarNameChange,
+  downloadRobotFile,
+  onSaveToCloud,
+  onModelerSelectionChange,
+  onModelerElementChanged,
+} from '../../../utils/sidebarFunctionality';
```

Comment:
Let's please define a consistent naming scheme for the methods. Maybe we talk about handlers for all methods or how do we get that named consistently?